src/emysql_conv.erl

```diff
@@ -78,3 +82,13 @@ json_val({datetime,{ {Year,Month,Day}, {Hour,Min,Sec} }}) ->
                       [Year, Month, Day, Hour, Min, Sec]));
 json_val(Value) ->
     Value.
+
+-ifdef(maps_available).
+%% @see emysql:as_maps/1
+as_maps(#result_packet { rows = Rows } = Result) ->
+    Fields = emysql:field_names(Result),
+    [begin
+         maps:from_list([{binary_to_atom(K, utf8), V} || {K, V} <- lists:zip(Fields, Row)])
```

Comment:
Line 91 is too long:          maps:from_list([{binary_to_atom(K, utf8), V} || {K, V} <- lists:zip(Fields, Row)]).